SSPSolution/SSPSolution/LevelState.cpp

```diff
@@ -81,10 +81,8 @@ int LevelState::Initialize(GameStateHandler * gsh, ComponentHandler* cHandler, C
 	playerP->PC_AABB.ext[0] = 1.5;
 	playerP->PC_AABB.ext[1] = 1.5;
 	playerP->PC_AABB.ext[2] = 1.5;
-	playerG->worldMatrix = DirectX::XMMatrixIdentity();	//FIX THIS
-	this->m_player1.Initialize();
-	this->m_player1.SetGraphicsComponent(playerG);
-	this->m_player1.SetPhysicsComponent(playerP);
+	playerG->worldMatrix = DirectX::XMMatrixIdentity();		//FIX THIS
```

Comment:
A quick fix to graphic components which lack a constructor/initialization of world matrix. Nothing that needs to be fixed in this pull request.